src/import/pull-common.c

```diff
@@ -149,8 +151,21 @@ int pull_make_local_copy(const char *final, const char *image_root, const char *
         return 0;
 }
 
+static int hash_url(const char *url, char **ret) {
+        uint64_t h;
+        static const sd_id128_t k = SD_ID128_MAKE(df,89,16,87,01,cc,42,30,98,ab,4a,19,a6,a5,63,4f);
+
+        assert(url);
+
+        siphash24((uint8_t *) &h, url, strlen(url), k.bytes);
+        if (asprintf(ret, PRIx64, h) < 0)
```

Comment:
Yep, thanks.

This won't fix the "error: initializer element is not constant" though. And I have really no idea why that won't compile on the semaphore instance. I don't know how to reproduce that in my environment, so I can only guess...